src/CMakeLists.txt

```diff
@@ -48,12 +48,13 @@ ENDIF()
 # Main library.
 ADD_LIBRARY(roboptim-core SHARED ${HEADERS} ${ROBOPTIM_CORE_SRC})
 TARGET_COMPILE_OPTIONS(roboptim-core PUBLIC "-Droboptim_core_EXPORTS")
-PKG_CONFIG_USE_DEPENDENCY(roboptim-core eigen3)
 PKG_CONFIG_USE_DEPENDENCY(roboptim-core liblog4cxx)
 
 # Add required libs to pkg-config file.
 SET(ROBOPTIM_API_BOOST_LIBRARIES date_time system filesystem)
-PKG_CONFIG_APPEND_BOOST_LIBS(${ROBOPTIM_API_BOOST_LIBRARIES})
+IF(NOT WIN32)
+  PKG_CONFIG_APPEND_BOOST_LIBS(${ROBOPTIM_API_BOOST_LIBRARIES})
+ENDIF(NOT WIN32)
```

Comment:
Sorry, small correction for the previous message :
It works on windows iff the path to the boost libraries is included.

Because of the 2 last modifs on cmake/boost.cmake, it is required to add this 'else' condition to make it work on windows for plugins (it will add the path, but not the libs, as desired).

...
ELSE(NOT WIN32)
 PKG_CONFIG_APPEND_BOOST_LIBS()
ENDIF(NOT WIN32)